main.go

```diff
@@ -8,17 +8,25 @@ import (
 	"path"
 )
 
+func isY(text string) bool {
+	return text == "y\n" || text == "y\r\n"
```

Comment:
I think a better solution would be just to trim any leading whitespace characters from the string and simply compare to `"y"` or `"n"`. Can we implement that instead?

Reply:
Sure. I didn't realize that `TrimSpace` can trim any space without additional magic, when I first tried.

I've amended the commit.